packages/components/src/components/content-expander/content-expander.jsx

```diff
@@ -35,6 +34,10 @@ const ContentExpander = ({
         }
     }, [is_visible]);
 
+    React.useEffect(() => {
+        toggleVisibility(is_expanded);
+    }, [is_expanded]);
+
```

Comment:
(question) Is there a use-case for changing `is_expanded` prop from the parent after the component is mounted?

Reply:
There is a check added here in the PR for the number of accounts the user has. The value of `is_expanded` is true by default but is changed to false if the max number of accounts are exceeded

Reply:
I see the point. Thanks. Whenever you are free, can we add a comment for the purpose of the code?

Reply:
Yeah sure thing 👍